shared/utility/motion/splines/SmoothSpline.hpp

```diff
@@ -90,7 +90,7 @@ namespace utility::motion::splines {
         /**
          * Inherit Load Points
          */
-        virtual void importCallBack() override {
+        void importCallBack() override {
```

Comment:
I would wonder why virtual was here in the first place.

Reply:
If it's using override, it's virtual.
The clang-tidy rule that is being applied is if it's override the virtual does nothing